cloud/aws/templates/aws_oidc/bin/aws_cli.py

```diff
@@ -109,6 +109,23 @@ def wait_for_ecs_service_healthy(self):
             )
             time.sleep(30)
 
+    def fix_digest_value(self, value):
+        """
+        Sets the lock file digest value in DynamoDB to the given value.
```

Comment:
**suggestion (blocking):** would be helpful to mention that the digest is a chucksum of the terraform state file